spring-cloud-loadbalancer/src/main/java/org/springframework/cloud/loadbalancer/core/XForwardedHeadersTransformer.java

```diff
@@ -33,9 +33,9 @@
 
 public class XForwardedHeadersTransformer implements LoadBalancerClientRequestTransformer {
 
-	private final LoadBalancerClientFactory clientFactory;
+	private final ReactiveLoadBalancer.Factory<ServiceInstance> clientFactory;
 
-	public XForwardedHeadersTransformer(LoadBalancerClientFactory clientFactory) {
+	public XForwardedHeadersTransformer(ReactiveLoadBalancer.Factory<ServiceInstance> clientFactory) {
```

Comment:
Same here, changing the constructor

Reply:
Yes, it was just merged, not yet released :).